dependencies.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from database import SessionLocal
```

Comment:
The import `SessionLocal` is incorrect. It should be `async_session` based on the definition in `database.py` .